treeherder/config/settings.py

```diff
@@ -441,6 +441,9 @@
 
 # Only generate alerts for data newer than this time in seconds in perfherder
 PERFHERDER_ALERTS_MAX_AGE = timedelta(weeks=2)
+# From the same job's log, ingest (or not) multiple PERFHERDER_DATA dumps
+# pertaining to the same performance signature
+PERFHERDER_ENABLE_MULTIDATA_INGESTION = False
```

Comment:
This should be removed if the [separate PR](https://github.com/mozilla/treeherder/pull/6731/files) will be merged

Reply:
This PR was rebased on PR 6731.
That's why the changes *seem* to be duplicated here. But they're not duplicated.